script.js

```diff
@@ -1,81 +1,122 @@
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
     } else if (
       (humanChoice == "rock" && computerChoice == "scissor") ||
       (humanChoice == "paper" && computerChoice == "rock") ||
       (humanChoice == "scissor" && computerChoice == "paper")
     ) {
-      console.log(`You win! ${humanChoice} beats ${computerChoice}`);
+      message = `You win! ${humanChoice} beats ${computerChoice}`;
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
       humanScore++;
-      console.log(`Computer score: ${computerScore} \n\Human score: ${humanScore}`);
+      message = `Computer score: ${computerScore} \n\Human score: ${humanScore}`;
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
     } else if (humanChoice == computerChoice) {
-      console.log("Tie! None gets a point");
-    } else console.log("Error, combination not yet added");
+      message = "Tie! None gets a point";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    } else {
+      message = "Error, combination not yet added";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
   }
-  // Play game
-  const humanSelection = getHumanChoice();
+function playGame(humanButtonSelection) {
+  playRound();
+
+  const humanSelection = humanButtonSelection;
   const computerSelection = getComputerChoice();
-  // Print choices
-  console.log(`Computer chose ${computerSelection} \n\Human chose ${humanSelection}`);
+
+  let message = `Computer chose ${computerSelection} \n\Human chose ${humanSelection}`;
+  console.log(message);
+  gameData.classList.add("gameData");
+  gameData.textContent = message;
+  resultDiv.appendChild(gameData);
+
   playRound(humanSelection, computerSelection);
+  if (humanScore >= 5 || computerScore >= 5) {
+    if (humanScore > computerScore) {
+      message = "Human won the game, congratulations!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
+    else if (computerScore > humanScore) {
+      message = "Computer won the game, better luck next time human!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
+    else {
+      message = "It's a tie. Give it another go to get a winner!";
+      console.log(message);
+      gameData.classList.add("gameData");
+      gameData.textContent = message;
+      resultDiv.appendChild(gameData);
+    }
```

Comment:
You should be able to do this:

```suggestion
    if (humanScore > computerScore) {
      message = "Human won the game, congratulations!";
    }
    else if (computerScore > humanScore) {
      message = "Computer won the game, better luck next time human!";
    }
    else {
      message = "It's a tie. Give it another go to get a winner!";
    }
    
    console.log(message);
    gameData.classList.add("gameData");
    gameData.textContent = message;
    resultDiv.appendChild(gameData);
```

Reply:
or even creating a function like so:

```js
function logMessage(msg){
    console.log(message);
	  gameData.classList.add("gameData");
	  gameData.textContent = message;
	  resultDiv.appendChild(gameData);
}
```